src/CMakeLists.txt

```diff
@@ -7,7 +7,7 @@ add_library(sqlpp11-connector-sqlite3
 		detail/connection_handle.cpp
       )
 
-if (SQLPP_DYNAMIC_LOADING)
+if (DYNAMIC_LOADING)
```

Comment:
Why did you remove the prefix?

Reply:
There was a mismatch between the `SQLPP_DYNAMIC_LOADING` compile flag, and the cmake option. I forgot to put it back.

Fixed in latest commit.